charts/vela-core/crds/core.oam.dev_policydefinitions.yaml

```
@@ -15,7 +15,7 @@ spec:
     listKind: PolicyDefinitionList
     plural: policydefinitions
     shortNames:
-    - policy
+    - policy-def
```

```suggestion
    - def-policy
```

Start from def is better